packages/core/helper-plugin/src/content-manager/utils/getAttributeInfos.ts

```diff
@@ -0,0 +1,8 @@
+import get from 'lodash/get';
+
+const getType = <T extends object>(schema: T, attrName: string) =>
+  get(schema, ['attributes', attrName, 'type'], '');
```

Comment:
@Convly do we have a type for Schemas yet we could use?

CC @markkaylor

Reply:
We do, but for now, everything is located in strapi/strapi (which is not a dependency of the helper plugin if I recall correctly). We've started looking at solutions to create shared access to types without circular dependencies with Alex, but for now, this is still in ideation & brainstorm